src/gc/gcinterface.h

```diff
@@ -868,6 +868,11 @@ class IGCHeap {
     // Unregisters a frozen segment.
     virtual void UnregisterFrozenSegment(segment_handle seg) = 0;
 
+#if defined(FEATURE_BASICFREEZE) && defined(VERIFY_HEAP)
```

Comment:
There should be no ifdefs in the GCInterface interface. Provide implementation instead.

Reply:
right

Reply:
Done.